cpu/esp_common/Makefile.dep

```diff
@@ -44,7 +44,7 @@ ifneq (,$(filter esp_wifi_enterprise,$(USEMODULE)))
 endif
 
 ifneq (,$(filter netdev_default,$(USEMODULE)))
-  ifneq (,$(filter lwip,$(USEMODULE)))
+  ifneq (,$(filter lwip_arp,$(USEMODULE)))
```

Comment:
```suggestion
  ifneq (,$(filter lwip_netif,$(USEMODULE)))
```
`lwip_arp` isn't used with `LWIP_IPV6=1 LWIP_IPV4=0`.

Reply:
The problem only occurs with `LWIP_IPV4=1` (likely because of the `lwip_arp` module), `LWIP_IPV6=1` compiles fine with `esp_now`.